src/run.rs

```diff
@@ -0,0 +1,55 @@
+use chrono::prelude::*;
+use clap::ArgMatches;
+
+use crate::util::file::*;
+
+fn get_day(matches: &ArgMatches) -> Result<u32, std::num::ParseIntError>
+{
+    if let Some(day) = matches.get_one::<String>("day")
+    {
+        day.parse()
+    }
+    else
+    {
+        Ok(Utc::now().day())
+    }
+}
+
+pub async fn run(matches: &ArgMatches) -> Result<(), Box<dyn std::error::Error>>
+{
+    let day = get_day(matches)?;
+
+    if day < 1 || day > 25
```

Comment:
I guess this should be `day <= 1 || day >= 25` (or 0 and 26)? Otherwise it would fail on the first and last day

Reply:
nvm